### PR TITLE
Add Product ID in orders and query through product id

### PR DESCRIPTION
**Pull request by Matt-Kou**

### CODE REVIEW

File: service/static/js/rest_api.js [hunk 1]
```diff
@@ -141,13 +146,19 @@ $(function () {
...
             table += '</tr></thead><tbody>'
             let firstOrder = "";
             for (let i = 0; i < res.length; i++) {
                 let order = res[i];
-                table += `<tr id="row_${i}"><td>${order.id}</td><td>${order.customer_id}</td><td>${order.status}</td><td>${order.created_on}</td><td>${order.updated_on}</td></tr>`;
+                let item_ids = []
+                for (let j = 0; j < order.items.length; j++) {
+                    let item = order.items[j];
+                    item_ids.push(item.product_id);
+                }
```
Comment (MahamArif) [inline service/static/js/rest_api.js:160]: You can use a shorter syntax for this code:
```
item_product_ids = order.items.map(item => item.product_id)
```

This is much cleaner and simple. You can also join them via a separator to show on UI, as follows:
```
item_product_ids = order.items.map(item => item.product_id).join(',')
```

Comment (MahamArif) [reply service/static/js/rest_api.js:160]: Please provide a screenshot of the updated UI

File: service/static/index.html [hunk 2]
```diff
@@ -100,6 +108,7 @@ <h3>Create, Retrieve, Update, and Delete an Order:</h3>
             <th class="col-md-1">ID</th>
             <th class="col-md-3">Customer ID</th>
             <th class="col-md-3">Status</th>
+            <th class="col-md-3">Product ID</th>
```
Comment (MahamArif) [inline service/static/index.html:111]: rename this column to Product IDs, indicating multiple ids.

**Review by MahamArif — CHANGES_REQUESTED** [review]

File: service/static/js/rest_api.js [hunk 3]
```diff
@@ -151,12 +151,8 @@ $(function () {
             let firstOrder = "";
             for (let i = 0; i < res.length; i++) {
                 let order = res[i];
-                let item_ids = []
-                for (let j = 0; j < order.items.length; j++) {
-                    let item = order.items[j];
-                    item_ids.push(item.product_id);
-                }
-                table += `<tr id="row_${i}"><td>${order.id}</td><td>${order.customer_id}</td><td>${order.status}</td><td>${item_ids}</td><td>${order.created_on}</td><td>${order.updated_on}</td></tr>`;
+                item_product_ids = order.items.map(item => item.product_id).join(',')
```
Comment (MahamArif) [inline service/static/js/rest_api.js:154]: you missed a semicolon here.

**Review by MahamArif — CHANGES_REQUESTED** [review]

File: service/static/js/rest_api.js (hunk 3 above)
Comment (Matt-Kou) [reply service/static/js/rest_api.js:154]: It's not required to have semicolon in js I think

Comment (Matt-Kou) [reply service/static/js/rest_api.js:154]: Alright I added the semicolon to be more consistent

**Review by MahamArif — APPROVED** [review]

**Pull request merged**